beefy-gadget/src/gossip.rs

```diff
@@ -72,6 +72,9 @@ where
 		}
 	}
 
+	/// Note a live voting round
+	///
+	/// This should be called, if we cast a vote for `round`.
```

Comment:
It seems we actually call it upon receiving a vote as well (`handle_vote`).

Reply:
I actually meant that, because `handle_vote` is the only place `note_vote` is called. Will try to formulate the comment more generic.